packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -886,13 +887,13 @@ class ReactSixteenAdapter extends EnzymeAdapter {
       case ContextProvider || NaN: return 'ContextProvider';
       case Memo || NaN: {
         const nodeName = displayNameOfNode(node);
```

Comment:
One thing that wasn't clear to me was when/why we should use the adapter here or not.

Reply:
i think if we did, it would be an infinite loop.